service/src/main/resources/schema/development/example.json

```diff
@@ -0,0 +1,192 @@
+{
+  "dct:title": "sint",
```

Comment:
Since it wasn't obvious how to generate this example, can you document the steps to do it somewhere, such as in a file in the `docs` directory.